openmetadata-ui/src/main/resources/ui/src/pages/LogsViewer/LogsViewer.component.tsx

```diff
@@ -0,0 +1,312 @@
+    ) {
+      fetchMoreLogs();
+    }
+
+    if (toNumber(paging?.after) + 1 === toNumber(paging?.total)) {
+      // to stop at last page
+      setPaging({
+        ...paging,
+        after: undefined,
+      } as Paging);
+    }
+
+    return;
+  };
+
+  useLayoutEffect(() => {
+    const logBody = document.getElementsByClassName(
+      'ReactVirtualized__Grid'
+    )[0];
+
+    if (logBody) {
+      logBody.addEventListener('scroll', handleScroll, { passive: true });
+    }
+
+    return () => {
+      logBody && logBody.removeEventListener('scroll', handleScroll);
+    };
+  });
+
+  useLayoutEffect(() => {
+    const lazyLogSearchBarInput = document.getElementsByClassName(
+      'react-lazylog-searchbar-input'
+    )[0] as HTMLInputElement;
+
+    if (lazyLogSearchBarInput) {
+      lazyLogSearchBarInput.placeholder = `${t('label.search-logs')}...`;
+    }
+  });
+
+  const handleJumpToEnd = () => {
+    const logsBody = document.getElementsByClassName(
+      'ReactVirtualized__Grid'
+    )[0];
+
+    if (!isNil(logsBody)) {
+      logsBody.scrollTop = logsBody.scrollHeight;
+    }
+  };
+
+  const logSummaries = useMemo(() => {
+    return {
+      Type: ingestionDetails?.pipelineType || '--',
+      Schedule: ingestionDetails?.airflowConfig.scheduleInterval || '--',
+      ['Recent Runs']:
+        getIngestionStatuses(
+          ingestionDetails ? ingestionDetails : ({} as IngestionPipeline)
+        ) || '--',
+    };
+  }, [ingestionDetails]);
+
+  return isLoading ? (
+    <Loader />
+  ) : (
+    <div className="m-xs ">
+      {' '}
+      <div>
+        <Space
+          align="start"
+          className="w-full m-md m-t-xs"
+          direction="vertical">
+          <Space align="center">
+            <TitleBreadcrumb
+              titleLinks={getLogBreadCrumbs(
+                logEntityType,
+                ingestionName,
+                ingestionDetails
+              )}
+            />
+          </Space>
+          <Space>
+            <div>
+              <Typography.Title level={5}>
+                {ingestionDetails?.name}
+              </Typography.Title>
+            </div>
+          </Space>
+        </Space>
+
+        <Card className="h-full p-0 log-card">
+          {!isEmpty(logs) ? (
+            <Row>
+              <Col className="p-md" span={18}>
+                <Row className="relative">
+                  <Col span={24}>
+                    <Row justify="end">
+                      <div className="flex m-b-md">
+                        <Button
+                          ghost
+                          data-testid="jump-to-end-button"
+                          type="primary"
+                          onClick={handleJumpToEnd}>
+                          {t('label.jump-to-end')}
+                        </Button>
+
+                        <CopyToClipboardButton copyText={logs} />
+                      </div>
+                    </Row>
+                  </Col>
```

Comment:
we can avoid unnecessary wrapping, also <Row> must have <Col> as child

Reply:
@ShaileshParmar11 , Did not get this, can you suggest the changes?

Reply:
Thanks @ShaileshParmar11 .